mkt/settings.py

```diff
@@ -1001,8 +1001,13 @@ def JINJA_CONFIG():
 READ_ONLY = False
 
 REDIS_BACKENDS = {
```

Comment:
Yep, I need to come back to this. I think I will copy the single string parser that Django 1.7 removed into our codebase.